ReSharper.FSharp/src/FSharp.Psi.Features/src/CodeCompletion/FSharpScriptReferenceCompletionProvider.fs

```diff
@@ -44,10 +59,12 @@ type FSharpPathCompletionContextProvider() =
                 let caretOffset = caretOffset.Offset
                 let tokenOffset = token.GetTreeStartOffset().Offset
                 let tokenText = token.GetText()
-                let valueStartOffset = tokenOffset + tokenText.IndexOf("\"") + 1
+                let valueStart = tokenText.IndexOf("\"") + 1
+                let valueStartOffset = tokenOffset + valueStart
                 let tokenEndOffset = tokenOffset + tokenText.Length
 
                 let unfinishedLiteral = tokenText.IndexOf('"') = tokenText.LastIndexOf('"')
+                tokenText.IndexOf("nuget:", valueStart) = -1 &&
```

Comment:
What if completion is triggered at the beginning of the string? Usually we'd allow completing things that are applicable.
```fsharp
#r "{caret}nuget: Package"
```